rules/aws_vpc_flow_rules/aws_port_enumeration.py

```diff
@@ -0,0 +1,63 @@
+import ast
+import json
+from datetime import datetime, timedelta
+
+from panther_oss_helpers import get_string_set, put_string_set
+
+
+def get_key(event):
+    return __name__ + ":" + str(event.get("srcaddr"))
+
+
+def rule(event):
+    # VPC Flow logs have events where not all info are present
+    if (
+        not event.get("srcaddr")
+        or not event.get("dstaddr")
+        or not event.get("srcport")
+        or not event.get("dstport")
+    ):
+        return False
+
+    key = get_key(event)
+    accessed_ports = get_string_set(key)
+
+    # Mocking returns string - For testing only
+    if isinstance(accessed_ports, str):
+        if accessed_ports:
+            accessed_ports = ast.literal_eval(accessed_ports)
```

Comment:
I would be substantially more comfortable if we used `json.loads()` plus typecasting to a `set()` instead of `literal_eval()`.